Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1166,10 +1170,15 @@ def lhegensim(fragment,howMuch):
 steps['HLTDR2_2016']=merge( [ {'-s':'L1REPACK:Full,HLT:@%s'%hltKey2016,},{'--conditions':'auto:run2_hlt_relval'},{'--era' : 'Run2_2016'},steps['HLTD'] ] )
 steps['HLTDR2newL1repack_2016']=merge( [ {'-s':'L1REPACK:FullSimTP,HLT:@%s'%hltKey2016,},{'--conditions':'auto:run2_hlt_relval'},{'--era' : 'Run2_2016'},steps['HLTD'] ] )
 
+hltKey2017='relval2016' # replace this with 'frozen2017' after PR 19308 is merged
+menuR2_2017 = autoHLT[hltKey2017]
```

Comment:
@Martin-Grunewald 
I am going to set the menu for thie relval to `relval2017`
Objections ?